keps/sig-apps/4212-declarative-node-maintenance/kep.yaml

```diff
@@ -0,0 +1,62 @@
+title: Declarative Node Maintenance
+kep-number: 4212
+authors:
+  - "@atiratree"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-apps
+  - sig-autoscaling
+  - sig-cli
```

Comment:
We should also add `sig-cluster-lifecycle`? Seems @fabriziopandini and @sbueringer have been collaborating a lot and providing feedback, and they should be added to the reviewer list a well

Reply:
ack, updated the kep.yaml